src/interface/src/styleguide/scenario-card/scenario-card.component.ts

```diff
@@ -0,0 +1,99 @@
+import {
+  Component,
+  Input,
+  HostBinding,
+  Output,
+  EventEmitter,
+} from '@angular/core';
+import {
+  DatePipe,
+  CurrencyPipe,
+  NgIf,
+  NgSwitch,
+  NgClass,
+} from '@angular/common';
+import {
+  StatusChipComponent,
+  StatusChipStatus,
+} from '../status-chip/status-chip.component';
+import { ButtonComponent } from '../button/button.component';
+import { MatIconModule } from '@angular/material/icon';
+import { MatMenuModule } from '@angular/material/menu';
+import { MatButtonModule } from '@angular/material/button';
+export type ScenarioStatus = 'InProgress' | 'Running' | 'Done' | 'Failed';
+/**
+ * Scenario Card for displaying scenario data in a results list
+ */
+@Component({
+  selector: 'sg-scenario-card',
+  standalone: true,
+  imports: [
+    DatePipe,
+    NgIf,
+    CurrencyPipe,
+    NgSwitch,
+    NgClass,
+    StatusChipComponent,
+    ButtonComponent,
+    MatIconModule,
+    MatMenuModule,
+    MatButtonModule,
+  ],
+  templateUrl: './scenario-card.component.html',
+  styleUrl: './scenario-card.component.scss',
+})
+export class ScenarioCardComponent {
+  //TODO: maybe we need to accept the ID here, so we can emit events for these scenarios?
```

Comment:
I dont think we need an id... and for example if we are iterating we can do something like:

```  
 <sg-scenario-card *ngFor="let thing of things" (archiveScenario)="archiveScenario(thing.id)">

```

Where you thinking on another use case that might be better if the component has the id?

Reply:
Nope, I didn't have another use case in mind. You're right, this is the right approach.